content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -82,6 +80,14 @@ be down for the same period as the previous example (`08:29:00` to `10:21:00`,)
 The CronJob is only responsible for creating Jobs that match its schedule, and
 the Job in turn is responsible for the management of the Pods it represents.
 
+## New controller (alpha)
```

Comment:
```suggestion
## Choice of controller (alpha) {#cronjob-controller-version}
```

(Putting “alpha” in the fragment identifier is a problem because when the feature goes stable the URL of the fragment would change)

Reply:
Fair, removing.